modules/apps/feature-flag/feature-flag-web/src/main/java/com/liferay/feature/flag/web/internal/feature/flag/FeatureFlagsBag.java

```diff
@@ -83,17 +78,15 @@ public String getJSON() {
 	public boolean isEnabled(String key) {
 		FeatureFlag featureFlag = _featureFlagsMap.get(key);
 
-		if (featureFlag != null) {
-			return featureFlag.isEnabled();
-		}
-
-		_log.error(
-			StringBundler.concat(
+		if (featureFlag == null) {
+			String message = StringBundler.concat(
```

Comment:
@algobob there's no need to create variable for message this should just be passed in directly to the exception.

There also doesn't seem to be a good reason to change the control flow so it would be better to do less changes. Please move it back to the way it was with an early return of `featureFlag.isEnabled();` when `featureFlag != null`.